chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -130,6 +133,9 @@ data:
         proxy_set_header Authorization "Bearer $http_x_forwarded_access_token";
         {{- end }}
 
+        {{- if .Values.frontend.proxypassExtraSetHeader }}
+        proxy_set_header {{ .Values.frontend.proxypassExtraSetHeader }};
```

Comment:
I need this so that I can add an Authorization header that will be used by my API gateway when authenticating k8s service requests.

Reply:
Nice, glad that part works without issue :)